src/i18n/index.ts

```diff
@@ -0,0 +1,15 @@
+import bn from './bn/language.json' assert { type: 'json' };
+import en from './en/language.json' assert { type: 'json' };
+import es from './es/language.json' assert { type: 'json' };
+import fr from './fr/language.json' assert { type: 'json' };
+import id from './id/language.json' assert { type: 'json' };
+import it from './it/language.json' assert { type: 'json' };
+import ko from './ko/language.json' assert { type: 'json' };
+import pt from './pt/language.json' assert { type: 'json' };
+import th from './th/language.json' assert { type: 'json' };
+import tr from './tr/language.json' assert { type: 'json' };
+import uk from './uk/language.json' assert { type: 'json' };
+import vi from './vi/language.json' assert { type: 'json' };
+import zh from './zh/language.json' assert { type: 'json' };
```

Comment:
@chybisov Does it make sense to fix it that way, using that assert type json:

```
src/i18n/index.ts(1,16): error TS2732: Cannot find module './bn/language.json'. Consider using '--resolveJsonModule' to import module with '.json' extension.
src/i18n/index.ts(2,16): error TS2732: Cannot find module './en/language.json'. Consider using '--resolveJsonModule' to import module with '.json' extension.
src/i18n/index.ts(3,16): error TS2732: Cannot find module './es/language.json'. Consider using '--resolveJsonModule' to import module with '.json' extension.
src/i18n/index.ts(4,16): error TS2732: Cannot find module './fr/language.json'. Consider using '--resolveJsonModule' to import module with '.json' extension.
src/i18n/index.ts(5,16): error TS2732: Cannot find module './id/language.json'. Consider using '--resolveJsonModule' to import module with '.json' extension.
src/i18n/index.ts(6,16): error TS2732: Cannot find module './it/language.json'. Consider using '--resolveJsonModule' to import module with '.json' extension.
src/i18n/index.ts(7,16): error TS2732: Cannot find module './ko/language.json'. Consider using '--resolveJsonModule' to import module with '.json' extension.
src/i18n/index.ts(8,16): error TS2732: Cannot find module './pt/language.json'. Consider using '--resolveJsonModule' to import module with '.json' extension.
src/i18n/index.ts(9,16): error TS2732: Cannot find module './th/language.json'. Consider using '--resolveJsonModule' to import module with '.json' extension.
src/i18n/index.ts(10,16): error TS2732: Cannot find module './tr/language.json'. Consider using '--resolveJsonModule' to import module with '.json' extension.
src/i18n/index.ts(11,16): error TS2732: Cannot find module './uk/language.json'. Consider using '--resolveJsonModule' to import module with '.json' extension.
src/i18n/index.ts(12,16): error TS2732: Cannot find module './vi/language.json'. Consider using '--resolveJsonModule' to import module with '.json' extension.
src/i18n/index.ts(13,16): error TS2732: Cannot find module './zh/language.json'. Consider using '--resolveJsonModule' to import module with '.json' extension.
```

Reply:
@dennyscode should we use `"resolveJsonModule": true` is tsconfig as suggested in error?